APIs/schemas/token_schema.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Just noticed this says `draft-07`, whereas all current specs use `draft-04`, I believe?
If the newer features aren't absolutely required in IS-10, I suggest being consistent for now, and deciding in lock-step when we want to force implementations to adopt a more recent version - it looks like it might finally reach stable soon.